### DISCUSSION TITLE
Requested VAA not found in store (Solana WETH to WETH on Ethereum)

### DISCUSSION THREAD

**Discussion by Matt8white** [wormhole-foundation/wormhole#1515] [Problems / bugs] [answered]
Hello, i'm trying to transfer 1.26607386 Wrapped ETH from solana (7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs) to WETH on Ethereum but the process got stuck on step 3 "Waiting for Wormhole Network consensus..."

If I paste the tx ID in the recovery section in gives me the error in the title. (this is the tx on solana: 5PiqKmEPhLZafiybqGwJzAy264gdMAskBLndEyLDisF8u5hHvd7R8yFmvwWpXVpsE5bn61fW9PRQTgSHaF4gYZ6w)

What should I do to recover my 1,26 ETH?

**Comment by Matt8white** [comment]
Sorry, the transaction unstuck and ended correctly.

**Comment by wlunadev** [comment]
Having same issue (~3hr delay so far):
https://solscan.io/tx/4DdYrUrzQnfPPq9JrTeCMDT1ZFMwnbYFddJJs14Vf3efCwtEfad84FkTjyHfEpYuq1BQVSfuMu1JNBU74V6UNhHm

**Comment by evan-gray** [comment, marked as answer]
Since these posts, both of these transactions have been redeemed.
